### DISCUSSION TITLE
maestral didn't exists???

### DISCUSSION THREAD

**Discussion by metrolan** [Homebrew/discussions#2461] [Everyday usage] [answered]
I'm trying to install maestral https://formulae.brew.sh/cask/maestral#default but i can't, because there is no available formula ¿?

`brew install maestral`
`Warning: No available formula with the name "maestral". Did you mean marst?`
`==> Searching for similarly named formulae...`
`This similarly named formula was found:`
`marst`
`To install it, run:`
`  brew install marst`
`==> Searching for a previously deleted formula (in the last month)...`
`Error: No previously deleted formula found.`
`==> Searching taps on GitHub...`
`Error: No formulae found in taps.`

With brew install --cask maestral:

`brew install --cask maestral`
`Error: Cask 'maestral' is unavailable: No Cask with this name exists.`

Using Linuxmint 20.2, kernel 5.4.0-90-generic, python3 installed

Any help?

**Comment by carlocab** [comment, marked as answer]
Maestral is a cask, and casks are macOS-only, unfortunately. You should be able to install Maestral with `pip`, though.

**Reply by metrolan** [reply]
Ups... thank you.
